framework/src/system_dirs.ts

```diff
@@ -16,10 +16,10 @@ import * as path from 'path';
 
 // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types,@typescript-eslint/explicit-function-return-type
 export const systemDirs = (appLabel: string, rootPath: string) => ({
-	dataPath: path.join(rootPath, appLabel),
-	data: path.join(rootPath, appLabel, 'data'),
-	tmp: path.join(rootPath, appLabel, 'tmp'),
-	logs: path.join(rootPath, appLabel, 'logs'),
-	sockets: path.join(rootPath, appLabel, 'tmp', 'sockets'),
-	pids: path.join(rootPath, appLabel, 'tmp', 'pids'),
+	dataPath: path.resolve(rootPath, appLabel),
+	data: path.resolve(rootPath, appLabel, 'data'),
+	tmp: path.resolve(rootPath, appLabel, 'tmp'),
+	logs: path.resolve(rootPath, appLabel, 'logs'),
+	sockets: path.resolve(rootPath, appLabel, 'tmp', 'sockets'),
+	pids: path.resolve(rootPath, appLabel, 'tmp', 'pids'),
```

Comment:
Problem using this approach is that if ever `appLabel` got `/` in start it will skip the `rootPath` completely. e.g. 

```
path.resolve('/root/path', 'data', 'second/dir', '/third/dir`) 
```

The above will not result `/root/path/data/second/dir/third/dir` instead it will just return `/third/dir`.

So I would suggest to use `path.join` first and then use single value for `path.resolve` to get absolute path.

Reply:
@shuse2 any suggestions ?

Reply:
ah i also see the problem
i think we can do like
```
dataPath: path.resolve(path.join(rootPath, appLabel)),
```